diesel/src/sqlite/connection/raw.rs

```diff
@@ -194,6 +243,115 @@ extern "C" fn run_custom_function<F>(
     }
 }
 
+// Need a custom option type here, because the std lib one does not have guarantees about the discriminate values
+// See: https://github.com/rust-lang/rfcs/blob/master/text/2195-really-tagged-unions.md#opaque-tags
+#[repr(u8)]
+enum OptionalAggregator<A> {
+    // Discriminant is 0
+    #[allow(dead_code)]
```

Comment:
Is this needed anymore? I think we cover this case somewhere below in one of the match statements.

Reply:
The linter was telling me something about this case never being constructed. I will remove it and check again 👍